isPrime.js

```diff
@@ -31,6 +31,18 @@ console.log(result) // false
 
 **/
 
+
 function isPrime(num) {
-	// Your code
-}
\ No newline at end of file
+	for ( i = 2; i <= num; i++){//Check all number less than input num
```

Comment:
you don't have to check numbers up to the input number. checking up to the square root of the input is sufficient. e. g. when you check if 18 is divisible by 3, you automatically check for 6 as well since 3x6 = 18. so if you check up to sqrt(18) + 1 ~ 5, you are done. this would be cut down the running time of the code.